readinglist/tests/test_views_article.py

```diff
@@ -15,6 +15,18 @@ def setUp(self):
         self.before = resp.json
         self.url = '/articles/{id}'.format(id=self.before['id'])
 
+    def test_all_views_does_not_accept_basic_auth_when_deactivated(self):
+        headers = {'Authorization': 'Basic YWJjOmFi',
+                   'Content-Type': 'application/json'}
+        self.app.get('/articles', status=401, headers=headers)
+        self.app.post_json('/articles', MINIMALIST_ARTICLE, status=401,
+                           headers=headers)
+
+        self.app.get(self.url, status=401, headers=headers)
+        self.app.patch_json(self.url, MINIMALIST_ARTICLE, status=401,
+                            headers=headers)
+        self.app.delete(self.url, status=401, headers=headers)
+
```

Comment:
I suggest we move this test outside modification (``IntegrationTest(BaseWebTest, TestCase)`` ?)